logset.go

```diff
@@ -61,20 +70,23 @@ type LogSetReadRequest struct {
 }
 
 type LogSetReadResponse struct {
-	LogSet LogSet
+	LogSet
 	ApiResponse
 }
 
 func (l *LogSetClient) Read(readRequest LogSetReadRequest) (*LogSet, error) {
-	userClient := NewUserClient(l.AccountKey)
-	response, err := userClient.Read(UserReadRequest{})
+	resp, err := l.Get("/hosts/" + readRequest.Key)
```

Comment:
As a general remark the existing api will go away as we are working on offering a rest api.

As it is rather bad form to expose any of account/host/log keys, I would recommend using `POST` instead of a get to get the information, just to reduce the amount of data logged in systems (proxies, web server logs etc.).

We guarantee that a create_host with the same name will return the same host key for a given account key.